### ISSUE TITLE
Can't stop an activity from overview window

### ISSUE THREAD

**Issue by sixtyfive** [projecthamster/hamster#258]
So I rejoiced at a working Hamster too soon. Just tried to stop an activity for the first time and discovered that there's no way (no button, no context menu item) to do so. Do I need to compile a different branch instead of master or should this work somehow?

**Comment by toupeira** [comment]
Ah yes this isn't exactly too obvious: in the main window you can stop an activity by either entering an end time (i.e. change "14:00 foo" to "14:00 16:00 foo") or by starting another activity. You can also use the CLI command `hamster stop`. But really Hamster is meant to be used through an [extension like the one for Gnome-Shell](https://github.com/projecthamster/shell-extension) which includes a "Stop activity" entry in the dropdown menu.

I'm keeping this issue open though, a context menu in the main window would make a lot of sense.

**Comment by sixtyfive** [comment]
Thank you for the explanation!

Indeed, I would use Cinnamon's applet, but:

```
error t=2015-10-07T20:16:07.012Z [Applet "hamster@projecthamster.wordpress.com"]: Error importing applet.js from hamster@projecthamster.wordpress.com
error t=2015-10-07T20:16:07.013Z Could not load applet hamster@projecthamster.wordpress.com
```

**Comment by toupeira** [comment]
@sixtyfive sorry I'm not familiar with the Cinnamon extension, but it looks like you're hitting https://github.com/projecthamster/cinnamon-applet/issues/14

**Comment by ederag** [comment]
It is now [easier to stop an activity](https://github.com/projecthamster/hamster/issues/310#issuecomment-455236628)
but this could be improved; the `ActionRow` seems very promising:
https://github.com/projecthamster/hamster/blob/83edea9222acae426f8285999744f9462a2bdd03/src/hamster/widgets/facttree.py#L228-L229
(Not visible because commented out)

**Comment by ederag** [comment]
With PR #387 and specifically 5c321307b0ecf5e67295ea4591e5e0dcfab27819,
it is possible to use `Ctrl+Space` to stop the on-going activity.
Leaving open, waiting for a single mouse-click way to achieve that too.

**Comment by GeraldJansen** [comment]
Resolved by #427.

**Comment by ederag** [comment]
Indeed, now it is also possible to stop an activity by a single mouse click, thanks @GeraldJansen !
